index.js

```diff
@@ -30,14 +31,16 @@ const StatusBarShape = {
 };
 
 function customizeStatusBar(data) {
-  if (data.style) {
-    StatusBarIOS.setStyle(data.style, true);
-  }
-  const animation = data.hidden ?
+  if (Platform.OS === 'ios') {
+    if (data.style) {
+      StatusBarIOS.setStyle(data.style, true);
+    }
+    const animation = data.hidden ?
     (data.hideAnimation || NavigationBar.defaultProps.statusBar.hideAnimation) :
     (data.showAnimation || NavigationBar.defaultProps.statusBar.showAnimation);
 
-  StatusBarIOS.setHidden(data.hidden, animation);
+    StatusBarIOS.setHidden(data.hidden, animation);
+  }
 }
```

Comment:
Since that method has no `Android` code, can you make an early return, e.g:
```js
if (Platform.OS !== 'ios') return;
```
so that it's less nested?

Looks good apart from that. I guess that method should be `customiseStatusBarIOS` in that case since it's only iOS specific, CC: @Kureev